packages/ignite-dev-screens/templates/DevscreensButton.js

```diff
@@ -23,13 +23,18 @@ export default class DevscreensButton extends React.Component {
           <RoundedButton onPress={this.toggleModal}>
             Open DevScreens
           </RoundedButton>
-          <Modal visible={this.state.showModal} >
-            <PresentationScreen screenProps={{toggle: this.toggleModal}} />
+          <Modal
+            visible={this.state.showModal}
+            onRequestClose={
+              Platform.OS === 'android' ? this.toggleModal : undefined
```

Comment:
I don't think the Platform check is necessary. Any platforms that support a back button will call this function and those that don't won't.